owl/EASE-ACT.owl

```diff
@@ -409,6 +569,46 @@ One could argue Mental actions are all Physical actions, because anything the Ag
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-ACT.owl#MetaCognitionEvaluationTopic -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-ACT.owl#MetaCognitionEvaluationTopic">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-ACT.owl#MetaCognitionTopic"/>
+        <ease:ELANName>ta-meta-cognition-evaluation</ease:ELANName>
```

Comment:
Maybe it would be better to introduce new ELAN ontology module for the ELAN-specific definitions? Since these are more specific then the general comm definitions.

Reply:
I am ok with doing it later